test/fixture/main.tf

```diff
@@ -6,17 +6,45 @@ resource "random_id" "rg_name" {
   byte_length = 8
 }
 
+resource "azurerm_resource_group" "test" {
+  name     = "example-lb-${random_id.rg_name.hex}"
+  location = "West Europe"
+}
+
 module "mylb" {
-  source              = "../../"
-  resource_group_name = "${random_id.rg_name.hex}"
-  location            = "${var.location}"
-  prefix              = "${random_id.rg_name.hex}"
+  source                                 = "../.."
+  resource_group_name                    = azurerm_resource_group.test.name
+  type                                   = "private"
+  frontend_subnet_id                     = module.network.vnet_subnets[0]
+  frontend_private_ip_address_allocation = "Static"
+  frontend_private_ip_address            = "10.0.1.6"
+  lb_sku                                 = "Standard"
 
   remote_port = {
     ssh = ["Tcp", "22"]
   }
 
   lb_port = {
-    http = ["80", "Tcp", "80"]
+    http   = ["80", "Tcp", "80"]
+    https  = ["443", "Tcp", "443"]
+    https2 = ["1443", "Tcp", "1443", "/", "Https"]
```

Comment:
The same question as above about the mis-match between 3 parameters mentioned in doc and 5 parameters here

Reply:
On fact, I have changed the description in variables.tf.